src/game/object_helpers.c

```diff
@@ -1880,30 +1882,31 @@ void bhv_init_room(void) {
     o->oRoom = get_room_at_pos(o->oPosX, o->oPosY, o->oPosZ);
 }
 
-s32 is_mario_in_room(void) {
+s32 cur_obj_is_mario_in_room(void) {
```

Comment:
can you revert the re-naming here? its a better name but EZ upgradability 2.1 bla bla bla (or define the old name to this func for compat)

Reply:
That function is new in 2.1 so it won't matter when it's squashed.

Reply:
Either way nobody is using the old name yet

Reply:
yeah we can keep it then, i thought it was old